mlrun/api/crud/client_spec.py

```diff
@@ -31,6 +31,7 @@ def get_client_spec(self):
             mpijob_crd_version=mpijob_crd_version,
             ui_url=config.resolve_ui_url(),
             artifact_path=config.artifact_path,
+            default_feature_store_data_prefix=config.feature_store.data_prefixes.default,
```

Comment:
Also, I would suggest placing the entire `config.feature_store.data_prefixes` dictionary here, not just the default value. The UI today only creates parquet/nosql targets, IIRC, but it's not necessarily going to be the case going forward. Even today the `redisnosql` target uses a different prefix that is not the same as the default - if the UI will shift to using it (which actually may be the correct thing to do for CE) then this will not be enough.